src/common/models/customization/customization.ts

```diff
@@ -0,0 +1,11 @@
+export interface Customization {
+  headerBackground?: string;
+  svgIcon?: string;
+  hideGitHubIcon?: boolean;
+  externalView?: ExternalView;
+}
+
+export interface ExternalView {
+  name: string;
```

Comment:
I would call this 'title' I like to use title for things that are rendered and 'name' for internal ids